src/Particular.Approvals/Particular.Approvals.csproj

```
@@ -1,13 +1,13 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFrameworks>net452;netstandard2.0</TargetFrameworks>
+    <TargetFrameworks>net462;netstandard2.0</TargetFrameworks>
```

If we're updating this and not worrying about older release branches, we might as well bump this up to `net472`.

```suggestion
    <TargetFrameworks>net472;netstandard2.0</TargetFrameworks>
```